### PR TITLE
APIS-6449: Brought in domain libraries and aligned with changes in APM

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

**Review by AndySpaven — APPROVED** [review]

File: app/views/UpdateTermsAndConditionsLocationView.scala.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.developers.DeveloperSession
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.views.BackButtonWithJavaScript
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.routes
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/UpdateTermsAndConditionsLocationView.scala.html:21]: Same

File: app/views/checkpages/applicationcheck/team/TeamView.scala.html [hunk 2]
```diff
@@ -21,10 +21,10 @@
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.views.Link
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.applications.Application
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.developers.DeveloperSession
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/checkpages/applicationcheck/team/TeamView.scala.html:24]: Here

File: app/views/checkpages/applicationcheck/team/TeamView.scala.html [hunk 3]
```diff
@@ -21,10 +21,10 @@
...
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.checkpages.routes
 @import uk.gov.hmrc.apiplatform.modules.common.domain.models.LaxEmailAddress
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/checkpages/applicationcheck/team/TeamView.scala.html:27]: And again

File: app/views/checkpages/checkyouranswers/team/TeamView.scala.html [hunk 4]
```diff
@@ -21,7 +21,7 @@
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.developers.DeveloperSession
 @import views.html.include.DevMain
 @import views.html.include.teamMembers.teamList
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/checkpages/checkyouranswers/team/TeamView.scala.html:24]: And again

File: app/views/include/teamMembers/teamList.scala.html [hunk 5]
```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/include/teamMembers/teamList.scala.html:17]: More

File: app/views/include/teamMembers/teamList.scala.html [hunk 6]
```diff
@@ -14,10 +14,10 @@
...
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.views.{AnchorStyle, Button}
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.developers.DeveloperSession
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/include/teamMembers/teamList.scala.html:20]: Again

File: app/views/manageTeamViews/ManageTeamView.scala.html [hunk 7]
```diff
@@ -21,7 +21,7 @@
 @import include.teamMembers._
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.controllers.ApplicationViewModel
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.controllers.FraudPreventionNavLinkViewModel
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
Comment (petekirby-ee) [inline app/views/manageTeamViews/ManageTeamView.scala.html:24]: Another

File: app/uk/gov/hmrc/apiplatform/modules/uplift/views/WeWillCheckYourAnswersView.scala.html [hunk 8]
```diff
@@ -18,7 +18,7 @@
 @import include._
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.controllers.Crumb
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.views.BackButtonWithJavaScript
-@import uk.gov.hmrc.apiplatform.modules.applications.domain.models.ApplicationId
+@import uk.gov.hmrc.apiplatform.modules.common.domain.models.UserId
```
[the remark this answers is not in the document]
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/apiplatform/modules/uplift/views/WeWillCheckYourAnswersView.scala.html]: done

**Pull request merged**